src/index.tsx

```diff
@@ -189,6 +190,7 @@ export const Netwatch: React.FC<IProps> = (props: IProps) => {
               nRequests={nRequests}
               connections={connections}
               clearAll={clearAll}
+              showStats={props.showStats ? true : false}
```

Comment:
why not just `showStats={props.showStats}` ?

Reply:
I tried but I had a typescript error

Reply:
Fix done